src/envoy/http/jwt_auth/config.proto

```diff
@@ -191,4 +191,11 @@ message AuthFilterConfig {
   //      path_exact: /healthz
   //
   repeated HttpPattern bypass_jwt = 3;
+
+  // From
+  // https://github.com/envoyproxy/data-plane-api/blob/11c750628bcaeeeca3f72f89fe5c99921aa727b1/envoy/config/filter/http/jwt_authn/v2alpha/config.proto
+  // If true, the request is allowed if JWT is missing or JWT verification
+  // fails. Default is false, a request without JWT or failed JWT verification
+  // is not allowed.
+  bool allow_missing_or_failed = 4;
```

Comment:
Btw, I noticed that this implementation does NOT reject if token is not there (at least from my observation when testing the policy). Not very important for our purpose, but you may want to rename and/or change the comment here to reflect the correct impact.

Reply:
I have changed the comment based on your observation.